.circleci/config.yml

```diff
@@ -14,7 +14,8 @@ jobs:
       - run:
           name: Create env and install dependencies
           command: |
-            conda env create -f environment.yml
+            conda install mamba -n base -c conda-forge
+            mamba env create -f environment.yml
```

Comment:
Is mamba as stable as mamba?  I heard that it is faster but don't knot much about it. Snakemake is using mamba as well.

Reply:
I think it's quite widely used. It doesn't hang all the time like Conda. It still uses conda-forge so the packages are the same.